lms/static/js/edxnotes/views/tab_view.js

```diff
@@ -116,9 +116,7 @@ define([
         showErrorMessage: function (message) {
             this.$('.wrapper-msg')
                 .removeClass('is-hidden')
-                .find('.msg-content .copy').text(message)
-                .closest('.msg')
-                .focus();
```

Comment:
FWIW I had suggested @talbs remove the focus since the user would have to navigate back _up_ to the search form (somewhat unintuitive). I suggested using a `required` attribute on the search field itself which would catch any empty queries and likely cover 99% of the cases until a more in-depth accessibility overhaul could be made.